generator/src/test/java/com/scottlogic/deg/generator/cucumber/utils/CucumberGenerationConfigSource.java

```diff
@@ -40,6 +40,11 @@ public CucumberGenerationConfigSource(CucumberTestState state) {
         return Collections.emptyList();
     }
 
+    @Override
+    public GenerationConfig.MonitorType getMonitorType() {
+        return GenerationConfig.MonitorType.QUIET;
```

Comment:
If you are hardcoding Quiet here there is no need to add a boolean to CucumberTestState.